### ISSUE TITLE
How to enable xray tracing 

### ISSUE THREAD

**Issue by matthias-pichler-warrify** [aws/aws-sdk-js-v3#1795]
**Describe the issue with documentation**

I think the documentation is missing information on how to enable tracing via XRAY for the v3 clients.

**To Reproduce (observed behavior)**

**Expected behavior**

There should be a section (probably towards the beginning) that explains how to use either the build in (if it exists 🤷 ) or `aws-xray-sdk-core` tracing

**Screenshots**

**Additional context**

**Comment by dradetsky** [comment]
On a related note, it would be great if you guys could get together with the aws-otel people and make a version of https://github.com/aspecto-io/opentelemetry-ext-js/tree/master/packages/plugin-aws-sdk compatible the v3 sdk. I don't want to feel like I can't upgrade because tracing will get worse.

**Comment by ffxsam** [comment]
I'm stuck on this too. If I try the usual way of doing this:

```js
const s3 = AWSXRay.captureAWSClient(
  new S3({ apiVersion: API_VERSION })
);
```

I get the error `TypeError: service.customizeRequests is not a function`.

**Comment by willarmiros** [comment]
The API for instrumenting clients is different for V3 of the AWS SDK than for V2. We have just added this support to the X-Ray SDK and the documentation can be found here: https://github.com/aws/aws-xray-sdk-node/tree/master/packages/core#capture-outgoing-aws-requests-on-a-single-client

Please note that as of 3/29/21, these changes are not yet released. They will be included in the next release, v3.3.0, of the X-Ray SDK. You can watch that repo to be notified of that release.

**Comment by ajredniwja** [comment]
Thanks for updating @willarmiros, I am gonna close this issue since the documentation is present which had to come from the x-ray SDK. 

Please @ffxsam, @matthias-pichler-warrify please reach out if you have any other questions.
